### PR TITLE
Force `composite_tree` to match config

### PR DESCRIPTION
**Pull request by uvsmtid-gs**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -128,6 +128,7 @@ def list_dir(
         "jsonschema",
         "flasgger",
         "marshmallow",
+        "marshmallow-oneofschema",
```
Comment (uvsmtid) [inline setup.py:131]: Was it helful? If not, remove.

File: src/argrelay/composite_tree/CompositeNodeType.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+
+class CompositeNodeType(Enum):
+    """
+    FS_33_76_82_84 composite tree node types.
+    """
+
+    init_node = auto()
+    """
+    TODO: Do we even need it?
```
Comment (uvsmtid) [inline src/argrelay/composite_tree/CompositeNodeType.py:11]: Address TODO-s in this file.

File: setup.py [hunk 3]
```diff
@@ -36,7 +36,7 @@ def list_dir(
 setuptools.setup(
     name = "argrelay",
     # See `docs/dev_notes/version_format.md`:
-    version = "0.6.13",
+    version = "0.6.14.dev0",
```
Comment (uvsmtid) [inline setup.py:39]: Finalize

File: src/argrelay/composite_tree/CompositeNode.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    See `CompositeNodeType`.
+    """
+
+    sub_tree: Union[dict[str, BaseNode], None] = field()
+
+
+@dataclass
+class RootNode(BaseNode):
+    """
+    TODO: describe
```
Comment (uvsmtid) [inline src/argrelay/composite_tree/CompositeNode.py:28]: TODO

File: src/argrelay/composite_tree/CompositeTreeHead.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import Union
+
+from argrelay.composite_tree.CompositeNode import BaseNode, RootNode
+
+
+@dataclass
+class CompositeTreeHead:
```
Comment (uvsmtid) [inline src/argrelay/composite_tree/CompositeTreeHead.py:10]: Rename to CompositeForest

File: src/argrelay/plugin_interp/InterpTreeInterp.py [hunk 6]
```diff
@@ -147,7 +147,7 @@ def propose_arg_completion(self) -> None:
                 if (
                     isinstance(x, str)
                     and
-                    x != default_tree_leaf_
+                    x != surrogate_node_id_
```
Comment (uvsmtid) [inline src/argrelay/plugin_interp/InterpTreeInterp.py:150]: Should it be `surrogate_tree_leaf_`?

File: setup.py (hunk 1 above)
Comment (uvsmtid-gs) [reply setup.py:131]: Helpful

File: src/argrelay/composite_tree/CompositeNodeType.py (hunk 2 above)
Comment (uvsmtid-gs) [reply src/argrelay/composite_tree/CompositeNodeType.py]: Init/Root node was removed.

File: src/argrelay/composite_tree/CompositeTreeHead.py (hunk 5 above)
Comment (uvsmtid-gs) [reply src/argrelay/composite_tree/CompositeTreeHead.py]: Renamed.

File: src/argrelay/plugin_interp/InterpTreeInterp.py (hunk 6 above)
Comment (uvsmtid-gs) [reply src/argrelay/plugin_interp/InterpTreeInterp.py:150]: No, `x` is a `node_id` (`dict` key, not `dict` value).

**Review by uvsmtid — APPROVED** [review]

**Pull request merged**